.github/workflows/hello.yml

```diff
@@ -0,0 +1,7 @@
+on: pull_request
```

Comment:
```suggestion
on:
  pull_request:
```

Reply:
AFAIK, both works. but yeah, your version is the one you see in the docs a lot, so I'll just commit the suggestion.